include/zephyr/storage/disk_access.h

```diff
@@ -100,6 +100,16 @@ int disk_access_read(const char *pdrv, uint8_t *data_buf,
 int disk_access_write(const char *pdrv, const uint8_t *data_buf,
 		      uint32_t start_sector, uint32_t num_sector);
 
+/**
+ * @brief erase data from disk
```

Comment:
One request here, otherwise I'm good with this change- we should document here that the erased data must read back as `0x0` or `0xFF`. This way we align with what the test checks, and specifically exclude a disk operation like DISCARD from being implemented to satisfy this API.

Reply:
Finally a chance to close the very first issue I submitted to Zephyr #24417 :)

Reply:
Heh. Yeah, still assigned to me, somehow. You can have it.